src/eckit/runtime/TaskInfo.cc

```diff
@@ -28,11 +28,9 @@ namespace eckit {
 
 //----------------------------------------------------------------------------------------------------------------------
 
-TaskInfo::TaskInfo() {
+TaskInfo::TaskInfo() : Padded<Info, 4096>{} {
     ASSERT(busy_ == false);
 
-    eckit::zero(*this);
```

Comment:
Do we lose anything by removing this? This seems likely to be a functional change?

Reply:
See comment about ctor.